react/javascript/src/components/gherkin/Attachment.tsx

```diff
@@ -13,10 +15,12 @@ const Attachment: React.FunctionComponent<IProps> = ({ attachment }) => {
     return image(attachment)
   } else if (attachment.mediaType.match(/^video\//)) {
     return video(attachment)
+  } else if (attachment.mediaType == 'text/x.cucumber.log+plain') {
+    return text(attachment, prettyANSI, true)
   } else if (attachment.mediaType.match(/^text\//)) {
-    return text(attachment, (s) => s)
+    return text(attachment, (s) => s, false)
   } else if (attachment.mediaType.match(/^application\/json/)) {
-    return text(attachment, prettyJSON())
+    return text(attachment, prettyJSON(), false)
```

Comment:
Can we change this from `prettyJSON()` to `prettyJSON` like I had done in my PR?

Reply:
yes sure :)